index.html

```diff
@@ -944,293 +646,49 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
-turn, Resource Timing should interface with Fetch registry to
-surface resource fetch events.</p>
-<p class="note">The above resource exclusion is at risk as currently only one
-implementation passes the <a
-href="https://wpt.fyi/results/resource-timing/no-entries-for-cross-origin-css-fetched.sub.html?label=experimental"
->related test</a>.</p>
-</li>
-<li>If the resource's <a>Request</a>'s
-<a data-cite="Fetch#concept-request-destination">destination</a> equals to
-"document", and the <a>Request</a> was not triggered by
-<a data-cite="HTML#process-the-iframe-attributes">process the iframe attributes</a> or
-<a data-cite="HTML#process-the-frame-attributes">process the frame attributes</a>,
-abort the remaining steps.</li>
-<li><dfn data-lt="step-create-object">Create a new
-<a>PerformanceResourceTiming</a> object</dfn> and set
-<a>entryType</a> to the DOMString <code>resource</code>.</li>
-<li>Immediately before the user agent starts to queue the resource
-for retrieval, record the <a>current time</a> in <a>startTime</a>,
-and set <a>nextHopProtocol</a> to the empty DOMString.</li>
-<li>Record the initiator of the resource in
-<a>initiatorType</a>.</li>
-<li>Record the <a data-cite="HTML#resolve-a-url">resolved URL</a>
-  of the requested resource in <a>name</a>.</li>
-<li><em>Fetch start:</em> <dfn data-lt="step-fetch-start">Immediately before a
-user agent starts the <a data-cite="FETCH#concept-fetch" data-lt=
-'fetch'>fetching process</a></dfn>, record the <a>current time</a>
-as <a>fetchStart</a>. Let <a>domainLookupStart</a>,
-<a>domainLookupEnd</a>, <a>connectStart</a> and <a>connectEnd</a>
-be the same value as <a>fetchStart</a>.</li>
-<li><dfn data-lt="step-collection-start">If the user agent is to
-reuse the data from another existing or completed <a data-cite=
-"FETCH#concept-fetch">fetch</a> initiated from the <a>current
-document</a></dfn>, abort the remaining steps.</li>
-<li>If there is an <a data-cite=
-"service-workers-1#dfn-containing-service-worker-registration">active
-worker</a> ([[service-workers-1]]) matching the current browsing or
-worker context's, immediately before the user agent <a data-cite=
-"service-workers-1#service-worker-concept">runs the worker</a>
-record the time as <a>workerStart</a>, or if the worker is already
-available, immediately before the <a data-cite=
-"service-workers-1#on-fetch-request-algorithm">event named `fetch`
-is fired</a> at the active worker record the time as
-<a>workerStart</a>. Otherwise, if there is no matching
-<a data-cite="service-workers-1#dfn-service-worker-registration">service
-worker registration</a>, set <a>workerStart</a> value to zero.</li>
-<li>If the resource fails the <a>timing allow check</a> algorithm, the user
-agent MUST run the following substeps:
-  <ol>
-    <li>If the resource's <a>Request</a>'s
-    <a data-cite="Fetch#concept-request-destination">destination</a> equals to
-    "document", set <a>workerStart</a> to zero.
```

Comment:
Here I believe the Fetch integration resulted in setting workerStart to zero for all destinations. I think it's cleaner, but it's a behavior change so we want to make sure implementations align with that.

We should file bugs and add tests on that front.

Reply:
/cc @wanderview in case he has opinions given [past discussion](https://bugs.chromium.org/p/chromium/issues/detail?id=1026404#c13)

Reply:
Can you open a Resource Timing issue on the fact that we need better tests here to verify the new behavior? (as well as reporting that as an implementation issues)

Reply:
Added https://github.com/w3c/resource-timing/issues/264